suggestions/service.py

```diff
@@ -0,0 +1,83 @@
+import abc
+import logging
+
+from mailjet_rest import Client
+
+from .model import Suggestion
+
+
+class SuggestionService(abc.ABC):
+    """
+    Service to send a suggestion
+    """
+
+    @abc.abstractmethod
+    def add_suggestion(self, suggestion: Suggestion):
+        """
+        Method to send a suggestion
+        """
+
+
+class MailjetSuggestionService(SuggestionService):
+    def __init__(
+        self,
+        mailjet_client: Client,
+        suggestion_email_address: str,
+        suggestion_name: str,
+        suggestion_mailjet_custom_id: str,
+    ):
+        self.mailjet_client = mailjet_client
+        self.suggestion_email_address = suggestion_email_address
+        self.suggestion_name = suggestion_name
+        self.suggestion_mailjet_custom_id = suggestion_mailjet_custom_id
+        self.logger = logging.getLogger(__name__)
+
+    def add_suggestion(self, suggestion: Suggestion):
+        data = {
+            "Messages": [
+                {
+                    "From": {
+                        "Email": suggestion.email_address,
+                        "Name": suggestion.name,
+                    },
+                    "To": [
+                        {
+                            "Email": self.suggestion_email_address,
+                            "Name": self.suggestion_name,
+                        }
+                    ],
+                    "Subject": suggestion.title,
+                    "TextPart": suggestion.content,
+                    "CustomID": self.suggestion_mailjet_custom_id,
+                }
+            ]
+        }
```

Comment:
Reading into mailjet's documentation I found that senders must be active and validated. Therefore, both sender and recipient should be known :/

We could include `suggestion.email_address` into the email body.

Reply:
Also, `suggestion_email_address` and `suggestion.email_address` are kind of similar. After this change what do you think of names like `email_sender`, `email_recipient` and keep `suggestion.email_address`?